__tests__/app.test.js

```diff
@@ -21,8 +22,27 @@ describe("GET /api", () => {
       .get("/api")
       .expect(200)
       .then(({ body: { endpoints } }) => {
-        console.log()
         expect(endpoints).toEqual(endpointsJson);
       });
   });
 });
+
+describe("GET /api/topics", () => {
+  test("200: Responds with an array of topic objects, each of which should have slug and description properties:", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body: { topics } }) => {
+        expect(topics).toEqual(topics_test);
+      });
+  });
+  test("404: Responds with a not found error msg for when an attempt to use an valid path that does not exist", () => {
+    return request(app)
+      .get("/api/topicsssss")
```

Comment:
Good test to check that the endpoint exists, you could be more direct by testing against an endpoint like /api/not-a-valid-endpoint, just to be clear to someone who may be looking over your test suite, but this does work.